analyzers/its/regression-test.ps1

```diff
@@ -226,6 +226,26 @@ function Show-DiffResults() {
     } -errorMessage $errorMsg
 }
 
+function Invoke-JsonParser()
+{
+    $jsonParser = Join-Path $PSScriptRoot "..\src\ITs.JsonParser\"
+    $arguments = @(
+        "run"
+        "--project"
+        $jsonParser
+        "--" # separates 'dotnet run' arguments from the command-line arguments of the program.
```

Comment:
Why do you need `dotnet run`? This only adds indirection that doesn't seem to be needed

Reply:
How would you do it?
Right now I'm calling `dotnet run`  on the `csproj` directory, so I build+run the project in one go.

Reply:
Discussed offline, I applied the suggestion